File: bench/tasks/07-functions/go/main.go
package main

import "fmt"

func greet(name string) string {
	return "Hello, " + name
}

func square(number int) int {
	return number * number
}

func main() {
	fmt.Println(greet("friend"))
	fmt.Println("Six squared is " + fmt.Sprint(square(6)))
}

File: bench/tasks/07-functions/rust/main.rs
fn greet(name: &str) -> String {
    format!("Hello, {}", name)
}

fn square(number: i32) -> i32 {
    number * number
}

fn main() {
    println!("{}", greet("friend"));
    println!("Six squared is {}", square(6));
}
